### PR TITLE
Add recommendation to respect Data Saver mode

### PR DESCRIPTION
**Pull request by mugdhalakhani**
and Battery Saver mode.

### CODE REVIEW

**Comment by mugdhalakhani** [comment]
@beverloo PTAL.

File: README.md [hunk 1]
```diff
@@ -153,8 +154,13 @@ The interfaces mimic regular Background Sync, but substitute `navigator.sync` wi
 ## Power and Network conditions
 The original explainer allowed the website to specify power and network conditions under which the
 periodic task should be run. This proposal leaves those decisions to the browser implementation,
-because we believe the browser is better positioned to make those choices for the user. This also
-makes for a simpler interface for developers.
+because we believe the browser is better positioned to make those choices for the user.
+This also makes for a simpler interface for developers.
+
+It is recommended however, that the browser respect user indications such as Data Saver and Battery
+Saver modes and skip sending periodicsync events when the user has turned these on. This would also
```
Comment (beverloo) [inline README.md]: `periodicsync`, not periodicsync

Comment (beverloo) [inline README.md]: Data Saver and Battery Saver, while descriptive, isn't the cross-platform terminology to use due to the capitalization. You could rephrase as:

> When the device is in either battery or data saving mode, the `periodicsync` event SHOULD not be fired.

That also adds some rfc2119ness :).

File: README.md [hunk 2]
```diff
@@ -153,8 +154,13 @@ The interfaces mimic regular Background Sync, but substitute `navigator.sync` wi
...
+cover situations where the browser incorrectly detects whether the device is connected to an
+unmetered connection.
```
Comment (beverloo) [inline README.md]: _some_ situations, not _all_ situations

**Review by beverloo — APPROVED** [review]
Thanks! LG % comments

Why is this repo on my user? Didn't we copy this to the WICG backgroundsync one?

**Pull request merged**